phoebe/frontend/bundle.py

```diff
@@ -3991,7 +3991,7 @@ def ld_coeffs_len(ld_func, ld_coeffs):
                                             True, 'run_compute')
                         else:
                             atm = self.get_value(qualifier='atm', component=component, compute=compute, context='compute', atm=kwargs.get('atm', None), **_skip_filter_checks)
-                            if atm not in ['ck2004', 'phoenix', 'tmap']:
+                            if atm not in ['ck2004', 'phoenix', 'tmap_sdO', 'tmap_DA', 'tmap_DAO', 'tmap_DO']:
```

Comment:
can we store this list somewhere in `passbands.py` and import it here?

Reply:
if the list appears anywhere else, then I think consolidating to one place would be useful to avoid them falling out of sync, otherwise this is fine for now

Reply:
This is the same as _supported_atms in passbands.py just with blackbody and extern_* removed.  It is also defined multiple times in passbands.py when checking for how to handle limb-darkening etc., so I think we could consolidate that everywhere and just have it defined once in passbands.py

Reply:
ok, if that doesn't fit here @aprsa - please just create a follow-up ticket so we don't forget to address it eventually

Reply:
I added it to the project, but I don't think we should roll this into this PR.

Reply:
Done; action item is [here](https://github.com/orgs/phoebe-project/projects/2/views/11?pane=issue&itemId=57136481).